libsolutil/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ set(sources
 )
 
 add_library(solutil ${sources})
-target_link_libraries(solutil PUBLIC jsoncpp Boost::boost Boost::filesystem Boost::system range-v3)
+target_link_libraries(solutil PUBLIC jsoncpp Boost::boost Boost::filesystem Boost::system range-v3 fmt::fmt-header-only)
```

Comment:
This and the URL-encoding helpers are things I'd put in a separate commit. They should stand out as its own thing in history.